product-features/farming/how-to.md

```diff
@@ -0,0 +1,208 @@
+
+## :tractor: Harvesting your farming rewards
+
+Farming will earn you rewards over time.
+
+At the end of every staking cycle (525 blocks, approximately 3.5 days), rewards will be available to harvest. To claim your rewards, follow these steps:
+
+1. Go to the [Farms page](https://app.alexlab.co/farm) on ALEX Lab App, which you can access through the Earn -> Farm tab.&#x20;
+2. You will find the **My Farms** panel which your active farms. Click on the farm you want to harvest, either from the panel or from the farm list.
+3. Expand the **My Farming** dashboard. If a cycle has ended, you will see rewards available to claim.
+4. Click on the "Harvest All" button and confirm the transaction on your wallet (just as you did in the previous guides).
+5. Wait for the transaction to be confirmed on the network. Remember, you can turn on [Telegram notifications](https://t.me/stacks\_tx\_notification\_bot) or search for the transaction on the [ALEX Explorer](https://app.alexlab.co/explorer).
+6. Once the transaction is completed, the reward amounts will be reflected in your wallet balance. You can always check your balance also on the ALEX Lab App, located beside the "Wallet Manager" at the top menu bar.
+
+You can collect these rewards and use them for various purposes on the ALEX Lab Platform. For example, you can [stake](https://app.alexlab.co/stake) your $ALEX rewards manually to generate compounding interest. You can also use your APower rewards to increase your access to IDOs on the [ALEX Launchpad](https://app.alexlab.co/launchpad). You can even buy more LP tokens!
+
+### How often should I harvest my rewards?
+
+To maximize your returns, it is best to harvest your rewards at the end of every cycle. This way, you have them available to generate more rewards! 🤩
+
+For example, you can manually stake your \$ALEX rewards to generate compounding interest. If your rewards are another token, you can still [swap](https://app.alexlab.co/swap) and convert them to \$ALEX. Another option would be to use your rewards to buy more LP tokens and benefit from being a [liquidity provider](../liquidity-providers/README.md).
+
+Happy Farming! 🥕 🥬 🍅
+
+## :bellhop: Withdrawing LP tokens (unstake)
+
+Withdrawing you LP tokens takes just a few steps. The important thing is when to do it.
+
+When farming, you are commit to locking up your tokens for a predefined period (reward cycles, each one ~3.5 days). Once these cycles conclude, you will be able to unstake them and regain control over your LP tokens.
+
+If you staked multiple times at different moments, the lock periods apply to each amount separately.
+
+Let's go through it step-by-step:
+
+1. Go to the [Farms page](https://app.alexlab.co/farm) on ALEX Lab App, which you can access through the Earn -> Farm tab.
+2. Find the farm from which you want to withdraw LP tokens and click on it.
+3. You will see the **My Farming** dashboard. Expand it to see all your farming details.
```

Comment:
2. You will see the **My Farming** dashboard. Expand it to see all your farming details.
3. Find the farm from which you want to withdraw LP tokens and click on it.